schedule/reminders.js

```diff
@@ -115,18 +115,19 @@ module.exports = {
         var db = options.db;
 
         // gets all clinics
-        db.medic.view('medic', 'clinic_by_phone', {
+        db.medic.view('medic-client', 'doc_by_type', {
```

Comment:
It's actually related - clinic_by_phone was broken due to the linked contacts changes IIRC so I either had to update it or replace it.